.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
```

Comment:
In relation to Git guardian and pre-commit checks, I am curious to find out why my commits always fail. I always have to add the -n flag to be able to commit. Attached is a screenshot which demonstrates these commit failures. @danielogen 

- https://docs.gitguardian.com/internal-repositories-monitoring/integrations/git_hooks/pre_commit?utm_source=product&utm_medium=GitHub_checks&utm_campaign=check_run_comment

<img width="1678" alt="Screenshot 2022-03-01 at 16 23 17" src="https://user-images.githubusercontent.com/1590213/156974259-99ac4741-0309-4466-9573-6f7a4961f866.png">

Reply:
@Baalmart , I am very sure. The `-n` flag is used to skip the precommit check. Let me test it out see.

Reply:
Thanks @danielogen , just wondering the relevance if I am going to always skip this check. 🤔 cc: @123MwanjeMike

Reply:
True @Baalmart , that wouldn't make sense. I guess there is something we are missing!

Reply:
I think we need to do more investigation into it

Reply:
To fix this, you need need to [create an API key](https://docs.gitguardian.com/internal-repositories-monitoring/gg_shield/getting_started#step-1-authenticate-using-an-api-key) and then add it to the **_.env_** file at the root of the directory (same location as .pre-commit-config.yaml) or in your **_~/.bash_profile_** file

_/.env_
`GITGUARDIAN_API_KEY=yourGeneratedAPIkeyHere`

 cc @Baalmart @danielogen

Reply:
let me try it out asap. Thanks @123MwanjeMike